docs/50-sdk.md

```diff
@@ -223,10 +224,204 @@ Find some samples and getting stated tutorial in the `examples/sdk/` directory o
         - str
 
 
+- `submit_job(self, job_configuration)`
+
+    Submit an AZTK Spark Job
+
+    Parameters:
+
+        - job_configuration: aztk.spark.models.JobConfiguration
+            The configuration of the job to be submitted
+    
+    Returns:
+        
+        - None
+
+- `list_jobs(self)`
+
+    List all created AZTK Spark Jobs
+
+    Parameters:
+
+        - job_configuration: aztk.spark.models.JobConfiguration
+            The configuration of the job to be submitted
+    
+    Returns:
+        
+        - List[aztk.spark.models.Job]
```

Comment:
So... this again is probably fine for now but once we introduce schedules what is this going to return? The list of jobs or the schedule? Or maybe the list of running and completed jobs and the time when it will run again?

Reply:
For list in particular, I think that this should probably still return AZTK Jobs. When we add scheduling, I think the AZTK Job model should be updated to include a list of all Batch Jobs that have run under it.

The CLI on list should display display:
1. The name of the JobSchedule
2. The state of the job schedule
3. The Job Schedule's creation time

On a get, the CLI should display all of the above plus the state of all Batch Jobs that have run under the Job Schedule.

We should provide the ability to get information & logs about the applications for each of those runs (batch jobs).